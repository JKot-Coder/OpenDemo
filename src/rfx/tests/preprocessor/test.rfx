// :RUN_RFXC -P %STD_OUTPUT% %INPUT%

#define FOO BAR /* */ qwe      		LINE

FOO