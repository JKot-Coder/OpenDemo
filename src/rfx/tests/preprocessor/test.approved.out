exit_status:0
output:
Source output:
    #line 1 "D:/Work/my/OpenDemo/src/rfx/tests/preprocessor/test.rfx"
    
    
    
    
    BAR qwe LINE
Diagnostic output:
